mycroft/client/enclosure/__init__.py

```diff
@@ -148,6 +148,7 @@ def process(self, data):
             self.ws.emit(Message("mycroft.disable.ssh"))
             self.ws.emit(Message("speak", {
                 'utterance': mycroft.dialog.get("reset to factory defaults")}))
+            time.sleep(5)
```

Comment:
why are we sleeping here an also have a method immediately followed called "wait_while_speaking"?